package.json

```diff
@@ -109,7 +109,6 @@
     "coveralls": "3.0.2",
     "enzyme": "3.3.0",
     "enzyme-adapter-react-16": "1.6.0",
-    "husky": "0.14.3",
```

Comment:
This is not a coveralls tool :)
`husky` is to integrate the git hooks to npm/yarn. The precommit hook will be useless if we remove this.

I think you meant to remove `"coveralls": "3.0.2",`